.github/workflows/ghcr.yml

```diff
@@ -149,7 +216,7 @@ jobs:
   test_runtime:
```

Comment:
When you say you are getting it unstuck, do you mean the job never runs sometimes? Or the job itself gets stuck?

Reply:
It just doesn't start, ever. Across multiple PR's.

Reply:
Ah interesting. It may be just because of the concurrency. Did you try just removing that at first?

In the builds that it never starts, do you see the `ghcr_build_runtime` job pass successfully?

Reply:
in this one from yesterday it all ran fine:
https://github.com/OpenDevin/OpenDevin/pull/3446

this one with latest "ok" commit from 7 hours ago is stuck:
https://github.com/OpenDevin/OpenDevin/pull/3410

idk what's going on tbh lol

Reply:
Close this PR now?